lldb/packages/Python/lldbsuite/test/tools/lldb-dap/lldbdap_testcase.py

```diff
@@ -357,6 +358,7 @@ def cleanup():
         self.addTearDownHook(cleanup)
         # Initialize and launch the program
         self.dap_server.request_initialize(sourceInitFile)
+        self.dap_server.request_configurationDone()
```

Comment:
Shouldn't we wait for `initialized` event?
```suggestion
        self.dap_server.wait_for_event("initialized")
        self.dap_server.request_configurationDone()
```